cupy_backends/cuda/libs/cusparselt.pyx

```diff
@@ -239,6 +260,12 @@ cpdef structuredDescriptorInit(Handle handle, MatDescriptor matDescr,
         <cusparseLtSparsity_t> sparsity)
     check_status(status)
 
+cpdef matDescriptorDestroy(MatDescriptor matDescr):
+    """Releases the resources used by an instance of a matrix descriptor."""
```

Comment:
If cuSPARSELt v0.0.1 is to be supported, we may want to add a guard in the newly added wrappers in this file:
```python
    if CUSPARSELT_VERSION < 100:
        raise RuntimeError(...)
```
See https://github.com/cupy/cupy/issues/4393.